actors/miner/tests/util.rs

```diff
@@ -900,54 +870,6 @@ impl ActorHarness {
         Ok(())
     }
 
-    pub fn confirm_sector_proofs_valid(
-        &self,
-        rt: &MockRuntime,
-        cfg: ProveCommitConfig,
-        pcs: Vec<SectorPreCommitOnChainInfo>,
-    ) -> Result<(), ActorError> {
-        let pieces = self.confirm_sector_proofs_valid_internal(rt, cfg.clone(), &pcs);
-
-        let mut all_sector_numbers = Vec::new();
-        for pc in pcs.clone() {
-            all_sector_numbers.push(pc.info.sector_number);
-        }
-
-        rt.set_caller(*POWER_ACTOR_CODE_ID, STORAGE_POWER_ACTOR_ADDR);
-        rt.expect_validate_caller_addr(vec![STORAGE_POWER_ACTOR_ADDR]);
-
-        let params = ConfirmSectorProofsParams {
-            sectors: all_sector_numbers,
-            reward_smoothed: self.epoch_reward_smooth.clone(),
-            reward_baseline_power: self.baseline_power.clone(),
-            quality_adj_power_smoothed: self.epoch_qa_power_smooth.clone(),
-        };
-
-        for sc in pcs.iter() {
-            if cfg.verify_deals_exit.contains_key(&sc.info.sector_number)
-                || cfg.claim_allocs_exit.contains_key(&sc.info.sector_number)
-            {
-                continue;
-            }
-            let unsealed_cid = sc.info.unsealed_cid.0;
-            let num = &sc.info.sector_number;
-            expect_sector_event(
-                rt,
-                "sector-activated",
-                &num,
-                unsealed_cid,
-                pieces.get(&num).unwrap(),
-            );
-        }
-
-        rt.call::<Actor>(
-            Method::ConfirmSectorProofsValid as u64,
-            IpldBlock::serialize_cbor(&params).unwrap(),
-        )?;
-        rt.verify();
-        Ok(())
-    }
-
     fn confirm_sector_proofs_valid_internal(
```

Comment:
I can see this helper is still used from the synchronous flow (only ProveCommitAggregate?), but please rename it to be more appropriate. Perhaps just expect_sectors_activated? Perhaps a comment explaining why it exists and can't be shared with prove_commit_sectors3.